pkg/kubectl/autoscale.go

```diff
@@ -38,6 +39,7 @@ func (HorizontalPodAutoscalerV1Beta1) ParamNames() []GeneratorParam {
 		{"scaleRef-kind", false},
 		{"scaleRef-name", false},
 		{"scaleRef-apiVersion", false},
+		{"scaleRef-apiGroup", false},
```

Comment:
@deads2k convinced me a separate field was worthwhile, since a common use will be specifying reference to a group+resource, but not caring about the version (or wanting the version to be selected via negotiation)